Broken test

Hello,

The test `tests/test_api.py::test_init_full` is currently broken:

```
======================================================================
FAIL: test_init_full (tests.test_api.TestTessBaseApi.test_init_full)
Test InitFull.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/build/tesserocr-2.8.0/tests/test_api.py", line 86, in test_init_full
    self.assertEqual(self._api.GetVariableAsString("file_type"), ".bl")
AssertionError: '.tif' != '.bl'
- .tif
+ .bl
``` 

This has been discovered while packaging this project in `nixpkgs`, a custom patch has been made as a workaround (https://github.com/NixOS/nixpkgs/pull/398949).

--- a/tests/test_api.py
+++ b/tests/test_api.py
@@ -83,7 +83,7 @@ class TestTessBaseApi(unittest.TestCase):
         self._api.End()
         self._api.InitFull(configs=configs, variables=vars_)
         # assert file_type from box.train and custom edges_childarea
-        self.assertEqual(self._api.GetVariableAsString("file_type"), ".bl")
+        self.assertEqual(self._api.GetVariableAsString("file_type"), ".tif")
         self.assertEqual(self._api.GetVariableAsString("edges_childarea"), "0.7")
         # reset back to default
         self._api.End()

